{{cookiecutter.project_slug}}/.github/workflows/publish.yml

```diff
@@ -0,0 +1,28 @@
+{% raw %}name: Publish
+
+on:
+  push:
+    branches:
+      - stable
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up Python
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.7
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install setuptools wheel
+    - name: Build package
+      run: |
+        python setup.py sdist bdist_wheel
+    - name: Publish to PyPI
+      uses: pypa/gh-action-pypi-publish@master
+      with:
+        user: {{ cookiecutter.pypi_username }}
```

Comment:
I think (¯\_(ツ)_/¯) it's possible to combine these so that the common block doesn't have to be written each time and the additional action is just a job dependency. It could be more bother than it's worth though.

Reply:
Discussed in person. For now, I am okay with duplicate jobs as they are pretty minimal. Something to look at for future for sure though.